File: src/Php/Writer/Csv.php
<?php
/**
 * @author   : Vitaliy Charnou <graffov87@gmail.com>
 */

declare(strict_types=1);

namespace Chervit\Iee\Writer;

class Csv
{
    const EXTENSIONS = 'csv';

    protected $file;

    public function setFile($file) :void
    {
        $this->file = $file;
    }

    public function getFile(): string
    {
        return $this->file;
    }

    public function setHeaders(array $headers)
    {
        $fp = fopen($this->file, 'w');
        fputcsv($fp, $headers);
        fclose($fp);
    }

    public function setInFile(array $item)
    {
        $fp = fopen($this->file, 'a+');
        fputcsv($fp, $item);
        fclose($fp);
    }
}

File: src/Python/Model/Writer/Csv.py
# @author   : Vitaliy Charnou <graffov87@gmail.com>
import csv


class Csv:
    __file = ''

    def setFile(self, file):
        self.__file = file

    def getFile(self):
        return self.__file

    def setHeaders(self, headers):
        with open(self.__file, "w") as csvFile:
            writer = csv.writer(csvFile)
            writer.writerow(headers)

    def setInFile(self, data):
        with open(self.__file, "a+") as csvFile:
            writer = csv.writer(csvFile)
            for line in data:
                writer.writerow(line)
